src/main/jib/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Plz, use unified `entrypoint.sh` with K8s support and docker secrets.

Reply:
this jib/entrypoint.sh is generated by jhipster for the gradle plugin com.google.cloud.tools.jib for local development. For travis buid I added /docker/Dockerfile and /docker/ entrypoint.sh

Reply:
@ishkurko